### DISCUSSION TITLE
如何无线投屏到小米电视等类似电视？

### DISCUSSION THREAD

**Discussion by tensor-tech** [linuxdeepin/developer-center#5221] [Q&A | 问答板块]
dde 如何 无线投屏整个屏幕呢？后续这块有没有开发？
gnome 推出了一个 https://flathub.org/apps/org.gnome.NetworkDisplays 希望参考，然后推出无线投屏，平常开会真的需要，栓Q

**Comment by BLumia** [comment]
v20 不太清楚，社区版 v23 上应该是可以满足你使用你所链接的那个应用的条件的（会支持 pipewire 且版本满足要求）。桌面环境程度的集成最早可能也要等到 v23 头一个正式版发布**后**才会考虑安排。

顺便 cc @meiyixiang

**Comment by Decodetalkers** [comment]
这个协议比较新，而且用的pipewire, gnome他们是制定协议的嘛。。需要时间研究

**Reply by tensor-tech** [reply]
协议是这个，https://en.wikipedia.org/wiki/Miracast
项目地址：https://gitlab.gnome.org/GNOME/gnome-network-displays
